catalyst-gateway/tests/schemathesis_tests/Earthfile

```diff
@@ -48,11 +54,10 @@ test-fuzzer-api:
         --load schemathesis:latest=(+package-schemathesis --api_spec=$api_spec --seed=$seed) \
         --load event-db:latest=(../../event-db+build) \
         --load cat-gateway:latest=(../+package-cat-gateway-integration) \
-        --service event-db \
```

Comment:
Why remove event-db?

Reply:
It's redundant and has no effect, because `cat-gateway` service itself depends on the `event-db` and `event-db` will run firstly in any case.